lib/stack-functions

```diff
@@ -106,14 +123,19 @@ stack-update() {
     echo "Could not find template (${template}). You can specify alternative template as second argument."
     return 1
   fi
+
   local params=$(_stack_params_arg $inputs)
   if [ -n "$params" ]; then local parameters="--parameters file://$params"; fi
 
+  local capabilities=''
+  local capabilities_value=$(_stack_capabilities $stack)
+  if [ "$capabilities_value" != "" ] ; then capabilities="--capabilities $capabilities_value" ; fi
```

Comment:
It's considered good style to write this as:
`[[ -z "$capabilities_value" ]] || capabilities="--capabilities $capabilities_value"`

We decided pretty early to appeal to the authority of google and adopt their style guidelines. You can find the rational at https://google.github.io/styleguide/shell.xml.

Reply:
Also there is a shell linter called `shellcheck` which I recommend too.

Reply:
If logic updated https://github.com/dushankw/bash-my-aws/commit/b3bf4a983ea52d76d073c0de805bfb7e93cdaf26

Regarding `shellcheck`, I know about it (it's actually a Syntastic linter in my vim). If you look, all of BMA has countless violations, fixing these is a much bigger piece.

I see nothing too egregious with what I have submitted, did you have a particular violation in mind?

Reply:
Nope. Not really. I guess I wrote most of bma before I discovered shellcheck. :)